src/components/EditProfile.vue

```diff
@@ -9,8 +9,9 @@
         <div class="modal-body">
             <label class="form-label">
               Profile Picture
-              <input type="file" class="form-control" @change="update($event.target.files)" accept="image/*">
+              <input type="file" class="form-control" @change="update" accept="image/*">
```

Comment:
Is there a way to restrict to a single file only? Could reduce the pain of multiple files trying to be uploaded.